sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -2480,3 +2480,41 @@ test_suite(
     tags = ["manual"],
     tests = ["sigverify_usage_constraint_{}".format(t["name"]) for t in test_cases],
 )
+
+opentitan_flash_binary(
+    name = "rom_ext_upgrade_test",
+    srcs = ["rom_ext_upgrade_test.c"],
+    devices = [
+        "fpga_cw310",
+    ],
+    signed = True,
+    deps = [
+        "//hw/ip/otp_ctrl/data:otp_ctrl_regs",
+        "//sw/device/lib/testing/test_framework:ottf_main",
+        "//sw/device/silicon_creator/lib:boot_data",
+        "//sw/device/silicon_creator/lib/drivers:lifecycle",
+        "//sw/device/silicon_creator/lib/drivers:otp",
+        "//sw/device/silicon_creator/lib/drivers:rstmgr",
+        "//sw/device/lib/testing/test_framework:ottf_ld_silicon_creator_slot_a",
+    ],
+)
+
+opentitan_functest(
+    name = "rom_ext_upgrade",
+    cw310 = cw310_params(
+    # We want to succeed once but not twice. First we want the rom_ext to start and change the minimume security level and then we want it to fail
+    exit_failure = "min_security_version_rom_ext:3|FAIL|min_security_version_rom_ext:0(?s:.*)min_security_version_rom_ext:0|PASS",
+        exit_success = "min_security_version_rom_ext:0(?s:.*)min_security_version_rom_ext:1(?s:.*)"+MSG_TEMPLATE_BFV.format(hex(CONST.BFV.BOOT_POLICY.ROLLBACK)),
+        test_cmds = [
+            "--exec=\"fpga load-bitstream --rom-kind=rom $(location {bitstream})\"",
```

Comment:
Nit: Replace ambiguous `location` with `rootpath`. https://bazel.build/reference/be/make-variables